src/pages/Login.tsx

```diff
@@ -38,18 +37,45 @@ function Login() {
                 alert("link didn't work");
             }, 1000);
         } else {
-            const randomSeed = randomString(100);
             const communication = new Communication();
-
-            communication.SSOWebsiteSendToMobile(randomSeed, requests);
-            setShowQR(randomSeed);
-
-            communication.onJwtToClient((data) => {
-                console.log(data);
-                window.location.replace(
-                    `/callback?requests=${data.requests}&accountName=${data.accountName}&username=nousername`
-                );
+            const logInMessage = new Message(jwtRequests[1]);
```

Comment:
requests should be VC, not Message. New Task

Reply:
agreed